pom.xml

```diff
@@ -51,10 +51,20 @@
   </dependencyManagement>
 
   <dependencies>
+
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-rx-java</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-rx-java2</artifactId>
```

Comment:
should be optional=true

Reply:
Same remark, we'd get `ClassNotFoundException` if the Jar was missing.

Reply:
does it mean that using vertx-junit5 impose to have rxjava on the classpath ?

Reply:
Yes

Reply:
if we could avoid that it would be better, however I understand that this is for test scope only and not compile scope

Reply:
We could lift this using reflection and type-specific classes. That's a more involving design.

Reply:
🤔  What if:

1. we merge this PR right now and have the functionality, and
2. record another issue where the focus is on making the 2 Rx lib dependencies optional?

Reply:
I think it's fine for now.

One option is to move it to vertx-rx library when the component stack will be decoupled from it (and there vertx-rx would optionally depends on vertx-junit5)